lib/screens/navigate/my_inventory.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:universal_io/io.dart';
+import '../navigate/navigate.dart';
+import 'package:flutter_excellence_hr/widgets/navigate/imp_notes_inventory.dart';
+import 'package:flutter_excellence_hr/resources/app_colors.dart';
+import 'package:flutter_excellence_hr/widgets/navigate/comment_section.dart';
+
+class YourInventory extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    if (Platform.isIOS) {
+      print('OS: ${Platform.isIOS}');
+    }
+ customDialog() {
+      return showDialog(
+          context: context,
+          builder: (BuildContext c) {
+            return Dialog(
+              shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(20)),
+              child: Container(
+                height: 450,
+                width: MediaQuery.of(context).size.width,
+                decoration: new BoxDecoration(
+                    borderRadius: BorderRadius.circular(20),
+                    color: AppColors.BACKGROUND_COLOR),
+                child: SingleChildScrollView(
+                  child: Column(children: <Widget>[
+                    Container(
+                      margin: EdgeInsets.fromLTRB(8, 16, 8, 16),
+                      child: Text(
+                        'Device Details',
+                        style: TextStyle(
+                          color: AppColors.LIGHTBLACK_COLOR,
+                          fontSize: 25,
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                    ),
+                    DropDown(),
+                    InventoryItemDetails(
```

Comment:
@Shaktiexcellence all Inventory item details need to be listviewbuilder

Reply:
Done it In ListView Builder